shuup/default_reports/reports/refunds.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2017, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.core.models import OrderLineType
+from shuup.default_reports.forms import OrderReportForm
+from shuup.default_reports.mixins import OrderReportMixin
+from shuup.reports.report import ShuupReportBase
+from shuup.utils.money import Money
+
+
+class RefundedSalesReport(OrderReportMixin, ShuupReportBase):
+    identifier = "refunded-sales"
+    title = _("Refunded Sales")
+    filename_template = "refunded-sales-%(time)s"
+    form_class = OrderReportForm
+
+    schema = [
+        {"key": "fully_refunded_orders", "title": _("Fully Refunded Orders")},
+        {"key": "partially_refunded_orders", "title": _("Partially Refunded Orders")},
+        {"key": "total_refunded", "title": _("Total Refunded")},
+    ]
+
+    def get_data(self, **kwargs):
+        orders = super(RefundedSalesReport, self).get_objects().filter(lines__type=OrderLineType.REFUND).distinct()
+
+        fully_refunded_orders = 0
+        partially_refunded_orders = 0
+        zero = total_refunded = Money(0, self.shop.currency)
+
+        for order in orders:
+            if order.get_total_unrefunded_amount() > zero:
+                partially_refunded_orders += 1
+            else:
+                fully_refunded_orders += 1
+
+            total_refunded += order.get_total_refunded_amount()
+
+        data = [
```

Comment:
not sure of the value of differentiating between partial and full refunds. IMO, should just query and annotate `OrderLine.refunds()`